src/senaite/core/browser/controlpanel/analysisprofiles/view.py

```diff
@@ -116,4 +124,7 @@ def folderitem(self, obj, item, index):
         obj = api.get_object(obj)
         item["replace"]["Title"] = get_link_for(obj)
         item["ProfileKey"] = obj.getProfileKey()
+        item["SampleTypes"] = obj.getRawSampleTypes()
```

Comment:
Please change this value to a comma separated list of titles of the sample types. Otherwise a list of UIDs will be shown when the table is exported.

Reply:
Good catch! Did not thought about the export!

Reply:
Done with https://github.com/senaite/senaite.core/pull/2509/commits/65bfefa953fb2ec04131b2f9cae9b9836b50ceb9